src/plugins/kibana_react/public/page_template/no_data_page/no_data_card/elastic_agent_card.tsx

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+/* eslint-disable @elastic/eui/href-or-on-click */
```

Comment:
Q: Is there a way that we can enforce this rule to the Cards created by this PR?
Maybe a runtime check?

Reply:
I'm actually disabling the rule here so that the cards **can** accept both in case of using react-router. Mainly I saw this in the Security Solution so I needed to continue to support their method.